conda/gateways/anaconda_client.py

```diff
@@ -44,11 +47,9 @@ def user_log_dir(self):
 
 def _get_binstar_token_directory():
     if "BINSTAR_CONFIG_DIR" in os.environ:
-        return EnvAppDirs(
-            "binstar", "ContinuumIO", os.environ["BINSTAR_CONFIG_DIR"]
-        ).user_data_dir
+        return os.path.join(os.environ["BINSTAR_CONFIG_DIR"], "data")
```

Comment:
test?

Reply:
Yeah, the whole module isn't covered I think, since we'll likely deprecate it in favor of non-token auth, I'm inclined to skip this